src/open_inwoner/pdc/tests/test_product.py

```diff
@@ -254,3 +254,33 @@ def test_sidemenu_button_is_rendered_when_no_cta_inside_product_content(self):
 
         self.assertTrue(sidemenu_cta_button)
         self.assertIn(product.link, sidemenu_cta_button[0].values())
+
+
+@override_settings(ROOT_URLCONF="open_inwoner.cms.tests.urls")
+class TestProductDetailView(WebTest):
+    def test_subheadings_in_sidebar(self):
+        product = ProductFactory(
+            content="##First subheading\rlorem ipsum...\r##Second subheading",
+            link="http://www.example.com",
+        )
+
+        response = self.app.get(
+            reverse("products:product_detail", kwargs={"slug": product.slug})
+        )
+
+        links = response.pyquery(".anchor-menu__sublist").find("a")
```

Comment:
Minor detail: you can expand the CSS selector instead of calling another `.find()`:

```py
links = response.pyquery(".anchor-menu__sublist a")
```